src/widgets/radio.jsx

```diff
@@ -19,6 +19,147 @@ var captureScratchpadTouchStart =
 
 var cx = React.addons.classSet;
 
+var Choice = React.createClass({
+    propTypes: {
+        checked: React.PropTypes.bool,
+        classSet: React.PropTypes.shape,
+        clue: React.PropTypes.shape,
+        content: React.PropTypes.shape,
```

Comment:
I think these three propTypes should be `React.PropTypes.object` (shape is for when you know what keys the object will have, and you can specify the types for those, such as: `React.PropTypes.shape({myKey: React.PropTypes.number.isRequired})`